io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/internal/CSVUtil.java

```diff
@@ -33,18 +34,33 @@
  */
 public class CSVUtil implements CSVConstants {
 
+//	/**
```

Comment:
There shouldn't be any line comments in the Javadoc comment

Reply:
solved